kit/popups/src/Tooltip/UITooltip.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import { Insets, Platform, Pressable, StyleSheet, TouchableOpacity } from 'react-native';
+import { makeStyles } from '@tonlabs/uikit.themes';
+import { UILayoutConstant } from '@tonlabs/uikit.layout';
+import type { UITooltipProps } from './types';
+import { UITooltipBox } from './UITooltipBox';
+
+const defaultHitSlop: Insets = {
+    top: -UILayoutConstant.smallContentOffset,
+    right: -UILayoutConstant.smallContentOffset,
+    bottom: -UILayoutConstant.smallContentOffset,
+    left: -UILayoutConstant.smallContentOffset,
+};
+
+function convertHitSlopToPaddings(hitSlop: Insets | undefined) {
+    if (!hitSlop) {
+        return defaultHitSlop;
+    }
+    return {
+        top: hitSlop.top !== undefined ? -hitSlop.top : defaultHitSlop.top,
+        right: hitSlop.right !== undefined ? -hitSlop.right : defaultHitSlop.right,
+        bottom: hitSlop.bottom !== undefined ? -hitSlop.bottom : defaultHitSlop.bottom,
+        left: hitSlop.left !== undefined ? -hitSlop.left : defaultHitSlop.left,
+    };
+}
+
+export function UITooltip({ children, style: styleProp, hitSlop, ...restProps }: UITooltipProps) {
+    const containerRef = React.useRef<TouchableOpacity>(null);
+    const [visible, setVisible] = React.useState(false);
+
+    const onClose = React.useCallback(function onClose() {
+        setVisible(false);
+    }, []);
+
+    const onOpen = React.useCallback(function onOpen() {
+        setVisible(true);
+    }, []);
+
+    const styles = useStyles(hitSlop);
+
+    /**
+     * Disabled `Pressable` is used here instead of `View` because the `onMeasure` method
+     * for the `ref` of the `View` does not return values on Android.
+     */
+    return (
+        <Pressable style={[styles.container, styleProp]} ref={containerRef} disabled>
+            {children}
+            {visible ? (
+                <UITooltipBox {...restProps} triggerRef={containerRef} onClose={onClose} />
+            ) : null}
+            <Pressable onPress={onOpen} style={styles.pressableOverlay} />
```

Comment:
Can you please repeat why we need pressable overlay? (It would be cool to leave a comment too)

Reply:
I left a comment

Reply:
We have to position this pressable overlay in an absolute position due to the fact that we need to implement hitSlop by paddings, and the total size of the component should remain unchanged (the same size as children has)

Reply:
It doesn't seem to work for android. Then I will shift the need to configure hitSlop with paddings to users.